scripts/test_layer2_disabled.py

```
@@ -0,0 +1,166 @@
+    # Get sections for a document (replace with actual document_id)
+    result = tool.execute_impl(
+        document_id="BZ_VR1",  # Example document
+        info_type="sections",
+    )
+
+    # Verify result
+    if result.success and result.data:
+        sections = result.data
+        print(f"\n✅ get_document_info returned {len(sections)} sections")
+        print(f"✅ Sample sections: {sections[:3]}")
+        print(f"✅ Test passed: section metadata available from layer3")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)